app/src/components/UI/Pages/BusinessGuides.tsx

```diff
@@ -1,14 +1,43 @@
-import { Container, Header, Card } from "semantic-ui-react";
+import { Container, Header, Card, Dropdown, Icon } from "semantic-ui-react";
 import HeaderBar from "../Molecules/HeaderBar/HeaderBar";
 import styles from "./businessGuide.module.css";
 
+const ProfileIcon = (
+  <span className={styles.user}>
+    <Icon name="user circle" size="big" /> Hi Colin
+  </span>
+);
+
+const options = [
+  {
+    key: "user",
+    text: (
+      <span>
+        Signed in as <strong>Colin Zhou</strong>
+      </span>
+    ),
+    disabled: true,
+  },
+  { key: "profile", text: "Profile Settings" },
+  { key: "page", text: "Create Vendor Page" },
+  { key: "help", text: "Help" },
+  { key: "sign-out", text: "Sign Out" },
```

Comment:
Since we used the phrase "Log in," could you change this to "Log out"?

Reply:
The scrollbar covering the menu was because there was no button in the top right hand corner, as it should. Once I added the button, it fixed the issue. I also changed it so it said "Log Out".

![image](https://user-images.githubusercontent.com/42430035/151020557-0b36de33-2bfc-4c21-93c2-33ac36178620.png)

Reply:
Also, I think it would be best if we finalize the HeaderBar component design right now, because I've already included this component in a lot of other files, and it will be a pain to change all of it again later...